tutorials/navigation/navigation_introduction_2d.rst

```diff
@@ -0,0 +1,147 @@
+.. _doc_navigation_overview_2d:
+
+Introduction
+===================
+
+Godot provides multiple objects, classes and servers to facilitate grid-based- or mesh-based navigation
+and pathfinding for 2D and 3D games. The following section provides a quick overview over all available
+navigation related objects in Godot for 2D scenes and their primary use.
```

Comment:
`object`s is used multiple times and strikes me as a bit odd compared to other Godot docs. Shouldn't we call them nodes and classes? Or are these different?